mepo.d/repository/git.py

```diff
@@ -239,6 +239,12 @@ def get_version(self):
             else:
                 name = tmp
                 tYpe = 'b'
+        elif output.startswith('HEAD'): # Assume hash
+            cmd = self.__git + ' rev-parse --short HEAD'
+            hash_out = shellcmd.run(cmd.split(), output=True)
+            detached = True
+            name = hash_out.rstrip()
+            tYpe = 'h'
```

Comment:
Is it "tYpe" or "type"?   Looks like a typo.

Reply:
I guess it's a typo, but it was Purnendu's original typo? I just copied and pasted. I can fix if you like.

Reply:
Well, it either is unused in which case it should be deleted.  Or it is a bug.  Probably worth figuring out which.